test/nodes/modelingtools.jl

```diff
@@ -0,0 +1,101 @@
+using ModelingToolkit
```

Comment:
@luap-pik : did you add this file on purpose?

Reply:
No :-)

Reply:
Actually I have no idea where it comes from :-D

Reply:
haha :D ;)